pkg/apis/storage/validation/validation.go

```diff
@@ -230,22 +229,6 @@ func ValidateVolumeAttachmentUpdate(new, old *storage.VolumeAttachment) field.Er
 
 var supportedVolumeBindingModes = sets.NewString(string(storage.VolumeBindingImmediate), string(storage.VolumeBindingWaitForFirstConsumer))
 
-// validateVolumeBindingMode tests that VolumeBindingMode specifies valid values.
-func validateVolumeBindingMode(mode *storage.VolumeBindingMode, fldPath *field.Path) field.ErrorList {
-	allErrs := field.ErrorList{}
-	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
-		if mode == nil {
-			allErrs = append(allErrs, field.Required(fldPath, ""))
-		} else if !supportedVolumeBindingModes.Has(string(*mode)) {
```

Comment:
don't drop validation completely, we still need to validate that if a value is specified, it is a supported value. we just want to remove the feature gate check from validation